participants/Pufferfish Cloud Storage Drive.md

```diff
@@ -0,0 +1,53 @@
+# <Pufferfish Cloud Storage Drive>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Project Name: Pufferfish Cloud Storage Drive, Our slack Handle: U017LEUM8HF
+Username: 2966696199
+
+## (1) Brief Project Description
+
+Puffer Cloud Storage Drive is a network cloud storage Drive, It's public open to any user from all over the world users. Who can sign up and get 8GB free storage space. Puffer Cloud storage different with traditional cloud storage provider like Dropbox ,Baidu Drive, it's use blockchain technology to make payment and use it in id identifications. It's physical storage based on Skynet storage. We glad to add Filecoin physical storage infrastructure to our user.
+We have three team work together and use different technique to access Filecoin storage pool. Now we'd like to use lotus and powergate attend SR2 competition. 
+
+## (2) Link to Project UI
+
+Who can use this web URL to access all features of Puffer Drive.
+http://www.pufferdrive.com
+
+## (3) What does your application/UI do?
+Puffer Drive front-end developed by Typescript + React + Ant Design 4.0 + OOP, User can use it's UI to upload/download/share/manage files. It's friendly.
+
+## (4) If your project is using a curated dataset, which one are you using?
+Our project currently didn't use any curated dataset, maybe after. We are using it for personal data cloud storage and public data storage and share.  
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+We have two data types, private and public. We don't have rights to access users' private data. We have exclusive rights to handle the shared public data. 
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We will invite our users to storage 100T+ public data during Slingshot competition.
+
+## (7) How are you structuring the data?
+All data is unstructured, we use hash to index them.
+
+## (8) What pre-processing are you doing before ingesting the data?
+We use database to storage and index all uploaded data. And provide public search feature. 
+
+## (9)  What tech stack will you use to store the data?
+We are planning to use lotus to access Filecoin storage physical infrastructure. Our another team will use Powergate.

```

Comment:
How will your users know that their data has been stored to Filecoin? How will you change your UI to allow retrieval directly from Filecoin?

Reply:
We will add new UI for this competition. Provide a public channel to upload competition data onto Filecoin & IPFS and index all those data for retrieval. 
A new search box will be added for users that want retrieval data.

Reply:
I think users didn't care where to store their data, we will keep them stored on Filecoin. Of course we can open hash value to user who want check their dataset stored on Firecoin & IPFS.